errbot/errbot.py

```diff
@@ -140,7 +140,7 @@ def get_config(config_path):
     return config
 
 
-if __name__ == "__main__":
```

Comment:
maybe leave the option to start it directly too ?
if __name__ == "__main__":
    main()

So we can still start it from the dev environment with no install ?